bin/mainflux-launch.sh

```diff
@@ -21,10 +21,7 @@ sleep 0.1
 mainflux-http &
 mainflux-manager &
 mainflux-writer &
-mainflux-coap &
```

Comment:
Why removing CoAP server? It is experimental, but still useful for many scenarios.

`mainflux-launch.sh` script is not a solution for professional deployment. It is for dev purposes also. In that case - let's keep CoAP server, and if developer does not need it - he needs to comment out this line.

Reply:
thanks for the tip @drasko 
I was wandering why it was removed from the Makefile with  #105 and it was still there in the launch script, so I thought we should keep the build and install script in sync.

I've added it back the launching of the CoAP server in the script. To make the things a bit clearer, I've also added small explanation how to build the CoAP server.

The explanation is also added to the docs with https://github.com/mainflux/doc/pull/13

Best Regards,
Jovan